docs/copilot/copilot-extensibility-overview.md

```diff
@@ -10,34 +10,46 @@ MetaSocialImage: images/shared/github-copilot-social.png
 ---
 # GitHub Copilot extensibility in VS Code
 
-GitHub Copilot is an AI coding assistant that helps you write code faster and with less effort. You can get code suggestions as you type in the editor, chat with Copilot to ask for help with your code, or get help with writing tests, fixing bugs, or refactoring code.
+You can extend GitHub Copilot in Visual Studio Code to expand the functionality of Copilot Chat or to take advantage of Copilot's Language Learning Model (LLM) to build AI features into your extension.
 
-Visual Studio Code is built with extensibility in mind. Almost every part of VS Code can be customized and enhanced through the Extension API. You can extend GitHub Copilot in your own Visual Studio Code extensions to enhance the experience for your users by adding AI-powered features to your extension. For example, let users ask domain-specific questions to your extension by using natural language, or combine large language models with your own business logic to provide intelligent code editing features, and much more.
+To extend Copilot in VS Code, you build a VS Code extension with additional capabilities, including:
 
-You have different options to extend GitHub Copilot in your VS Code extension:
+- Adding a chat participant in the VS Code Chat view to give users a natural-language interface
+- Using the Copilot LLM to interpret user prompts and generate responses
+- Using the Copilot LLM to implement AI-powered features in your extension
 
-- Use the [Chat API](/api/extension-guides/chat.md) to create a **chat participant** to give users a natural-language interface and provide domain-specific help via the Chat view in VS Code.
+## About Copilot extensibility in VS Code
 
-- Use the [**Language Model API**](/api/extension-guides/language-model.md) and the VS Code extension API to build AI-powered features into VS Code.
+You have different options to extend Copilot in your VS Code extension:
 
-With GitHub Copilot, you can expand the capabilities of your extension by using the power of AI and give developers a richer and more productive experience.
+- **VS Code Chat extension**: use the Chat API to create a chat participant that gives users a natural-language interface and provides domain-specific help via the Chat view in VS Code. Optionally, you can use the Copilot LLM to interpret user prompts and generate responses.
```

Comment:
I like these two. But feels like a bit of a repretition of the previous section. Not sure if we can de-duplicate this?